package.json

```diff
@@ -24,6 +27,7 @@
   },
   "dependencies": {
     "@architect/functions": "^5.3.3",
+    "@architect/plugin-lambda-invoker": "^0.1.1",
```

Comment:
1. [ ] Move this to devDependencies.
2. [ ] For now, install from your branch on GitHub for architect/plugin-lambda-invoker#2.